### ISSUE TITLE
Mulligan Guide always opens 1.0 version of deck

### ISSUE THREAD

**Issue by beheh** [HearthSim/Hearthstone-Deck-Tracker#3237]
We got a report on Discord that the Mulligan Button always opens the 1.0 version of a deck.

**Comment by azeier** [comment]
Works fine for me.

**Comment by azeier** [comment]
```
[7:47 PM] : i think it might've been specifically an issue with it updating automatically while I was in game, since it's working now
[7:48 PM] : I made an edit to the deck, it automatically put it to 1.1, and then clicking the button then took me to the 1.0 version
```

**Comment by azeier** [comment]
I think this should be fixed with v1.3
